src/module/item/flows/PackActionFlow.ts

```diff
@@ -46,10 +46,9 @@ export const PackActionFlow = {
      */
     async getPackActions(packName: string): Promise<SR5Item<'action'>[]> {
         console.debug(`Shadowrun 5e | Trying to fetch all actions from pack ${packName}`);
-        const pack = game.packs.find(pack => pack.metadata.system === SYSTEM_NAME && pack.metadata.name === packName);
+        const pack = game.packs.find(pack => pack.metadata.system === SYSTEM_NAME && pack.metadata.name === packName) as unknown as CompendiumCollection<'Item'> | undefined;
```

Comment:
do you need the `unknown`?

Reply:
Ups. Left that in after an earlier auto-complete that looked similar.